src/RA_Dlg_GameLibrary.cpp

```diff
@@ -286,11 +286,11 @@ void Dlg_GameLibrary::ThreadedScanProc()
             const DWORD nSize = ftell(pf);
             rewind(pf);
 
-            // May have caused a buffer overrun, this is way to big to be on the stack
-            auto pBuf{ std::make_unique<unsigned char[]>(6 * 1024 * 1024) };
+            auto pBuf = std::make_unique<unsigned char[]>(6 * 1024 * 1024);
 
             fread(static_cast<void*>(pBuf.get()), sizeof(unsigned char), nSize, pf); //Check
             Results.insert_or_assign(FilesToScan.front(), RAGenerateMD5(pBuf.get(), nSize));
+            pBuf.reset();
```

Comment:
In this case, it's unreasonable to put 6MB on the stack, so the change seems appropriate, but unnecessary. When pBuf is destructed it will also release the memory. But releasing early is always good, so I'd rather not fight the tool over this.